src/mumble/AudioInput.cpp

```diff
@@ -831,11 +839,11 @@ void AudioInput::selectNoiseCancel() {
 			iArg = 1;
 			break;
 		case Settings::NoiseCancelRNN:
-			qWarning("AudioInput: Using ReNameNoise as noise canceller");
+			qWarning("AudioInput: Using RNNoise as noise canceller");
```

Comment:
Please explicitly change this message to say ``RNNoise v0.2``. That way we will know what version users are using when they file bug reports.

Also, please change this to use a ``Log::info()`` instead of a ``qWarning``

Reply:
I assume log::info() is going to come from #6707, I'll wait for that to be merged in and rebase.

Reply:
For the time being you can use `qInfo()`, we will eventually replace all calls with the new ones in a dedicated PR.